packages/application/src/shell.ts

```diff
@@ -96,6 +99,7 @@ class ApplicationShell extends Widget {
     rightHandler.sideBar.addClass('jp-mod-right');
     rightHandler.stackedPanel.id = 'jp-right-stack';
 
+    bottomPanel.direction = 'left-to-right';
```

Comment:
Why change the layout direction to `left-to-right`? I had envisioned people possibly stacking things on top of the status bar.